README.adoc

```diff
@@ -951,7 +951,9 @@ end
 
 === Context Descriptions
 
-`context` block descriptions should always start with 'when', 'with', or 'without' and be in the form of a sentence with proper grammar when composed with `it` block descriptions.
+When an `it` block description is composed with a `context` block description, it should result in a sentence with
```

Comment:
I'm thinking about nested contexts, probably it makes sense to use the plural when talking about "context block description".